### PR TITLE
Work day fix

### PR DESCRIPTION
**Pull request by penright**
## Proposed change

I just added a note. If you look at bug report #31034 we were confused thinking that adding `holidays` to exclude meant to excluded holidays from consideration. I am not a grammar guru but I did tweak some. If I made any mistakes or it is not clear let me know.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31832
- This PR fixes or closes issue: 31034 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/workday.markdown [hunk 1]
```diff
@@ -66,31 +65,48 @@ add_holidays:
...
 <div class='note warning'>
 
 If you use the sensor for Norway (`NO`) you need to wrap `NO` in quotes or write the name in full.
-Otherwise the value is evaluated as `false`.
+Otherwise, the value is evaluated as `false`.
 If you use the sensor for Canada (`CA`) with Ontario (`ON`) as `province:` then you need to wrap `ON` in quotes.
-Otherwise the value is evaluated as `true` (check the YAML documentation for further details) and the sensor will not work.
+Otherwise, the value is evaluated as `true` (check the YAML documentation for further details) and the sensor will not work.
+
+> **_NOTE:_**
```
Comment (frenck) [inline source/_integrations/workday.markdown]: This is not how we do notes, besides it is a huge blob of text, which makes it hard to read.

Please check our developer documentation on how to make alerts/notes, and see if you can improve on this paragraph a bit to make it more readable.

File: source/_integrations/workday.markdown [hunk 2]
```diff
@@ -66,31 +65,48 @@ add_holidays:
...
 ```yaml
-# Example configuration.yaml entry
+# Example 1 configuration.yaml entry
+binary_sensor:
+  - platform: workday
+    country: US
+    workdays: [mon, tue, wed, thu, fri]
+    excludes: [sat, sun]
+```
+
```
Comment (frenck) [inline source/_integrations/workday.markdown:94]: Unneeded additional new line.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/workday.markdown [hunk 3]
```diff
@@ -66,31 +65,45 @@ add_holidays:
 {% endconfiguration %}
 
 Days are specified as follows: `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
-The keyword `holiday` is used for public holidays identified by the holidays module.
+The keyword `holiday` is used for public holidays identified by the holiday’s module.
```
Comment (springstan) [inline source/_integrations/workday.markdown]: The module is called `holidays`
```suggestion
The keyword `holiday` is used for public holidays identified by the holidays module.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**